kbfsgit/runner.go

```diff
@@ -358,6 +361,7 @@ func humanizeBytes(n int64, d int64) string {
 func (r *runner) printJournalStatus(
 	ctx context.Context, jServer *libkbfs.JournalServer, tlfID tlf.ID,
 	doneCh <-chan struct{}) {
+	r.printDoneIfNeeded("")
```

Comment:
I guess this means we won't get timing info or other profiles for the final index phase, when "-v" is passed to the git command one or more times?

Reply:
Hmmm good question. I'll need to test it.

Reply:
Yeah looks like we'd need to move `startTime`, `profName`, and `cpuProfFullPath` into the struct as well. Otherwise we are not writing memprof, or stopping CPU profiler. Do you want me to do that in this PR, or shall we fix it in go-git as you suggested?

Reply:
I think moving them into the struct makes sense, if you don't mind. Thanks!

Reply:
OK then; will do!